src/scss/_main.scss

```diff
@@ -971,6 +971,7 @@ nav {
 
   &.-failure,
   &.-error {
+    //here
```

Comment:
guessing this comment was left in by mistake?
```suggestion
```

Reply:
Removed.

Reply:
unless i'm mistaken i still see this in the latest changeset.